tests/shared/features/shared/step_definitions/test_steps.rb

```diff
@@ -24,6 +24,11 @@
   end
 end
 
+Then /^the step with an unsupported email header should raise:$/ do |step_string|
+  multiline_step = step_string.gsub(%{'''}, %{"""})
+  expect { steps(multiline_step) }.to raise_error(Spreewald::UnsupportedEmailHeader)
+end
```

Comment:
Sowas hätte ich eventuell noch generisch geschrieben, wie `/^the following steps should raise "(.?)":$/ do |exception_name, step_string|`

Reply:
Okay, beim nächsten Feature in der Art versuche ich entweder RSpec zu verwenden und den Test hier wieder auszubauen, oder ich ändere natürlich den Step um ihn wiederzuverwenden :+1: